utils/vqm2blif/test/scripts/test_vqm2blif.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
```

Comment:
It would be good to add an argument to this script to allow it to update the golden results, and document when to use that here.  Some changes to vqm2blif will change the output, so being able to quickly update the golden results and check the new ones in will be important.

Reply:
To match vtr_flow.py (script used for lots of flows), this argument would be named:
-create_golden

Reply:
Also useful to take a -help argument that prints out the usage of the script.

Reply:
We use argparse in most (all) of our python scripts, so copying its naming convention is probably best. https://docs.python.org/3/library/argparse.html
--help or -h should print something like:
usage: testvqm2blif.sh ...

Reply:
All of the requested features are now supported. Please refer to the attached images for sample functionality and a list of available option
![Screenshot from 2021-09-21 21-41-01](https://user-images.githubusercontent.com/25713466/134270898-c1cf733b-fec6-436b-828d-38cc977d39af.png)
![Screenshot from 2021-09-21 21-42-09](https://user-images.githubusercontent.com/25713466/134270902-4bad0e3d-eb11-4b76-8f65-ce6378888eb5.png)
s.